src/intTest/java/com/box/sdk/BoxRecentsIT.java

```diff
@@ -9,11 +11,13 @@
 import java.util.Iterator;
 import org.junit.AfterClass;
 import org.junit.BeforeClass;
+import org.junit.Ignore;
 import org.junit.Test;
 
 /**
  * {@link BoxRecents} related integration tests.
  */
+@Ignore
```

Comment:
I suppose that there is a problem with this test, right?

Reply:
I forgot to enable it. But it required user connection.